code/index.html

```diff
@@ -1,22 +1,229 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
+    <script src="https://use.fontawesome.com/bb61c0507d.js"></script>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <title>Dev News</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./assets/css/main.css" />
+    <meta name="description" content="A dev new website" />
+    <meta name="viewport" content="width=device-width,initial-scale=1.0" />
+    <script src="./app.js"></script>
```

Comment:
It can be really tricky to apply `script` in `<head />` - far more safe to place it at the end of `<body />`